edgexfoundry/app-functions-sdk-go
# feat(metrics): add http size to metrics, updates for new bootstrap version

apply to #1116

Signed-off-by: Elizabeth J Lee <elizabeth.j.lee@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) added http size to metrics
      <link to docs PR>

## Testing Instructions

1. Set debug logging for the services and run functional-tests, metrics-influxdb, and http-export with the configuration for http-export to write telemetry metrics for `HttpExportSize = true` 
2. Send data using postman to `localhost:59986/api/v2/resource/sample-numeric/int` and check that the metrics are printed to the log. 

## New Dependency Instructions (If applicable)
